ports/stm32/adc.c

```diff
@@ -135,7 +135,7 @@
 #elif defined(STM32H743xx)
 #define VBAT_DIV (4)
 #elif defined(STM32L475xx) || defined(STM32L476xx) || \
-      defined(STM32L496xx)
+      defined(STM32L496xx) || defined(STM32L432xx)
```

Comment:
Please can you insert the new addition in sorted order, so that would be first in the expression here.